setup.cfg

```diff
@@ -1,2 +1,36 @@
+[metadata]
+name = ctypeslib2
+author = Loic Jaquemet
```

Comment:
missing  version metadata
```suggestion
version="2.3.2",
author = Loic Jaquemet,
```

Reply:
It is by design. It is managed in git and populated by setuptools_scm.